lib/puppet/graph/simple_graph.rb

```diff
@@ -225,18 +225,18 @@ def report_cycles_in_graph
     return if n == 0
     s = n == 1 ? '' : 's'
 
-    message = "Found #{n} dependency cycle#{s}:\n"
+    message = _("Found %{num} dependency cycle%{s}:\n") % { num: n, s: s }
     cycles.each do |cycle|
       paths = paths_in_cycle(cycle)
       message += paths.map{ |path| '(' + path.join(" => ") + ')'}.join("\n") + "\n"
     end
 
     if Puppet[:graph] then
       filename = write_cycles_to_graph(cycles)
-      message += "Cycle graph written to #{filename}."
+      message += _("Cycle graph written to %{filename}.") % { filename: filename }
     else
-      message += "Try the '--graph' option and opening the "
-      message += "resulting '.dot' file in OmniGraffle or GraphViz"
+      message += _("Try the '--graph' option and opening the ")
+      message += _("resulting '.dot' file in OmniGraffle or GraphViz")
```

Comment:
I'm not sure if OmniGraffle or GraphViz should be translated - probably worth a translator note